scripts/kube_setup/03m_init_control_plane_and_copy_kube_conf.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+systemctl enable kubelet
+
+kubeadm config images pull
+
+# FIXME: Wildcard doesn't work
```

Comment:
@proman3419 create stories for fixme and todo

Reply:
https://github.com/k8loud/kubernetes-lab-setup/issues/14